UBUNTU2004-PG12.md

```diff
@@ -0,0 +1,97 @@
+# Configuring PostgreSQL 12 on Ubuntu 20.04
```

Comment:
Yes sure - happy to move it. I chose add to the root because the `/docs` directory appears to contain application info, not dev setup and config.

WDYT?